Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -27,16 +27,17 @@ import struct Utility.Toolchain
 import struct Utility.Path
 import PackageType
 
-let rootObjectReference =                           "__RootObject_"
-let rootBuildConfigurationListReference =           "___RootConfs_"
-let rootBuildConfigurationReference =               "_______Debug_"
-let rootGroupReference =                            "___RootGroup_"
-let productsGroupReference =                        "____Products_"
-let sourcesGroupReference =                         "_____Sources_"
-let testsGroupReference =                           "_______Tests_"
-let linkPhaseFileRefPrefix =                        "_LinkFileRef_"
-let sourceGroupFileRefPrefix =                      "__PBXFileRef_"
-let compilePhaseFileRefPrefix =                     "__src_cc_ref_"
+let rootObjectReference =                           "____RootObject_"
+let rootBuildConfigurationListReference =           "_____RootConfs_"
+let rootBuildConfigurationReference =               "_________Debug_"
+let rootGroupReference =                            "_____RootGroup_"
+let productsGroupReference =                        "______Products_"
+let sourcesGroupReference =                         "_______Sources_"
+let dependenciesGroupReference =                    "__Dependencies_"
```

Comment:
There is not a requirement to have a `_` prefix, so this diff can probably be less.